### PR TITLE
feat: add message when adding products into cart

### LINKED ISSUE #12: Add message
Show users a message for new products in cart.

### PR DESCRIPTION
**Pull request by LudovicaBolognin**

### CODE REVIEW

File: components/details/detail.html [hunk 1]
```diff
@@ -15,10 +15,23 @@ <h5 class="modal-title" id="staticBackdropLabel">Product: {{$ctrl.product[0].nam
...
+                        ng-click="$ctrl.onAddToCart($ctrl.product[0])">Add
                     to Cart
                 </button>
             </div>
         </div>
+        <div class="position-fixed bottom-0 end-0 p-3">
+            <div id="liveToast" class="toast hide" role="alert" aria-live="assertive" aria-atomic="true">
+                <div class="toast-header">
+                    <strong class="me-auto">Success</strong>
+                    <small>{{$ctrl.interval}} seconds ago</small>
```
Comment (mcelotti) [inline components/details/detail.html]: How many e-commerce websites did you use that had this idea of "added x seconds ago"? :) I'm just saying that maybe we could get rid of this "interval" feature...

File: components/details/detail.html [hunk 2]
```diff
@@ -15,10 +15,23 @@ <h5 class="modal-title" id="staticBackdropLabel">Product: {{$ctrl.product[0].nam
...
+                    <button type="button" class="btn-close" data-bs-dismiss="toast" aria-label="Close"></button>
+                </div>
+                <div class="toast-body">
+                    You add {{$ctrl.product[0].name}} into your cart.
```
Comment (mcelotti) [inline components/details/detail.html]: The message maybe could be "Added iphone to your cart"?

File: components/details/detail.html (hunk 1 above)
Comment (LudovicaBolognin) [reply components/details/detail.html]: I agree

**Review by mcelotti — APPROVED** [review]

File: components/details/details.controller.js [hunk 3]
```diff
@@ -12,6 +11,7 @@ const detailsController = function ($scope, productsService, $routeParams, $loca
 
   this.$onInit = () => {
     this.loadProduct();
+    this.startCounting = (new Date()).getSeconds();
```
Comment (mcelotti) [inline components/details/details.controller.js:14]: I think we forgot this, right?

Comment (LudovicaBolognin) [reply components/details/details.controller.js:14]: Thank you

**Pull request merged**